workshops/util.py

```diff
@@ -551,3 +551,32 @@ def update_event_attendance_from_tasks(event):
 
 def universal_date_format(date):
     return '{:%Y-%m-%d}'.format(date)
+
+
+def get_members(earliest, latest):
+    '''Get everyone who is a member of the Software Carpentry Foundation.'''
+
+    member_badge = Badge.objects.get(name='member')
+    instructor_badge = Badge.objects.get(name='instructor')
+    instructor_role = Role.objects.get(name='instructor')
+
+    # Everyone who is an explicit member.
+    explicit = Person.objects.filter(badges__in=[member_badge]).distinct()
+
+    # Everyone who qualifies by having taught recently.
+    implicit = Person.objects.filter(
+        task__role=instructor_role,
+        badges__in=[instructor_badge],
+        task__event__start__gte=earliest,
+        task__event__start__lte=latest
+    ).distinct()
+
+    # Merge the two sets.
+    return explicit | implicit
+
+
+def default_membership_cutoff():
+    "Calculate a default cutoff dates for members finding with `get_members`."
+    earliest = datetime.date.today() - 2 * datetime.timedelta(days=365)
```

Comment:
I can modify the default start date once I've checked the exact 
definition with the Steering Committee, but having a query parameter to 
set the "earliest" date allows us to get the answer even without that.